pysrc/bytewax/inputs.py

```diff
@@ -8,9 +8,7 @@
 import heapq
 from dataclasses import dataclass
 from typing import Any, Callable, Iterable, Tuple
-
-from .bytewax import AdvanceTo, Emit
-
+from .bytewax import AdvanceTo, Emit, KafkaInputConfig, ManualInputConfig, InputConfig
```

Comment:
The module of "input" is ballooning a bit but I pushed on modularizing more since we still only have the one "source"